agent/src/runner.py

```diff
@@ -199,4 +199,9 @@ def _load_routine_jobs(self) -> list[RoutineJob]:
                 cron_expression="0 12 * * 0",
                 logger=self._logger,
             ),
+            RoutineJob(
+                name="post_twitter",
+                cron_expression="0 9,12,18 * * *",
+                logger=self._logger,
+            ),
```

Comment:
Let's remove this for now - I want to first test this workflow in isolation before it's ready to be a RoutineJob